tests/unit/models/invite-test.js

```diff
@@ -76,11 +60,9 @@ describeModel(
                 ).to.equal(1);
 
                 expect(invite.email).to.equal('resend-test@example.com');
-                expect(
-                    invite.roles.length,
-                    'number of roles in request body'
-                ).to.equal(1);
-                expect(invite.roles[0], 'role ID').to.equal('1');
+                /* jscs:disable requireCamelCaseOrUpperCaseIdentifiers */
```

Comment:
Ghost-Admin is using `eslint` now so this will probably fail the lint tests when merged with master. `eslint` has a "next-line" mode to avoid needing to wrap single lines in disable/enable statements:

`// eslint-disable-next-line camelcase`